app/src/main/java/org/kiwix/kiwixmobile/settings/KiwixSettingsActivity.java

```diff
@@ -275,6 +275,11 @@ public void onSharedPreferenceChanged(SharedPreferences sharedPreferences, Strin
     }
 
     private void clearAllHistoryDialog() {
+      int warningResId;
+      if(nightMode(PreferenceManager.getDefaultSharedPreferences(getActivity())))
+        warningResId = R.drawable.ic_warning_white;
```

Comment:
Styling. Spacing is off. Also curtly brackets are very much preferred especially with an else stament.

Reply:
I'm sorry but I didn't quite understand what you meant by "Styling. Spacing is off"
Do you need space between <code>if</code> and <code>(</code> like we have in <code>onSharedPreferenceChanged</code> method ?

Reply:
Use  curly brackets for if-else statement.

Reply:
And the space as well. Consistent style is very important.